docs/changelog_for_devs.md

```diff
@@ -17,6 +19,8 @@
   Furthermore, there's a maximum swap fee, specified by the `swaps` pallet's
   on-chain constant `MaxSwapFee`.
```

Comment:
Isn't the addition of `zrml-styx` worth to mention? https://github.com/zeitgeistpm/zeitgeist/commit/1307e2a78b96d2603aca3497334132b62ebcc2de

Or has this file `changelog_for_devs` just the aim to inform the front end when something changes, which is already existing?

Reply:
You are right, we should include it. Added in [7783b04](https://github.com/zeitgeistpm/zeitgeist/pull/748/commits/7783b046196c9c8923c6dd53a150bb72fd20bb1f).